<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DecisionTreeForDatingClass</class>
 <widget class="QMainWindow" name="DecisionTreeForDatingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DecisionTreeForDating</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="chooseSampleButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>210</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>选择样本数据</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSample">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditInput">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="beginPredictButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>420</y>
      <width>181</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>开始预测</string>
    </property>
   </widget>
   <widget class="QPushButton" name="chooseInputButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>280</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>选择输入数据</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>381</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;4维特征：Weather，Temperature，Humidity，Wind&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;标签：Date（Yes/No）&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;利用决策树（ID3）和决策树（C4.5）分别进行预测&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>350</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>ID3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C4.5</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="drawingArea" native="true">
    <property name="geometry">
     <rect>
      <x>509</x>
      <y>19</y>
      <width>671</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="DecisionTreeForDating.qrc"/>
 </resources>
 <connections/>
</ui>
